packages/synapse-react-client/src/components/GoogleMap/GoogleMap.tsx

```diff
@@ -29,26 +29,30 @@ export type MapProps = {
 function Map(props: MapProps) {
   const { teamId, apiKeyOverride } = props
 
-  const { data: apiKey } = useQuery('googleMapsApiKey', getGoogleMapsApiKey, {
-    useErrorBoundary: true,
+  const { data: apiKey } = useQuery({
+    queryKey: ['googleMapsApiKey'],
+    queryFn: getGoogleMapsApiKey,
+    throwOnError: true,
   })
```

Comment:
The `useQuery` signature was changed so it now only takes one object argument which includes `queryKey` and `queryFn` properties.